seed/seed.json

```diff
@@ -500,7 +506,7 @@
                 {
                     "feature_association": {
                         "enable_feature": [
-                            "HangWatcher"
+                            "EnableHangWatcher"
```

Comment:
Thanks for fixing this.
`EnableHangWatcher` is the correct feature name.
Somehow `HangWatcher` was used in `main` by mistake.